src/tests/JIT/Generics/ConstrainedCall/static_virtual_calls.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+using System;
+using Xunit;
+
+
+public interface ITypeChecker
+{
+    static abstract bool Test<T>();
+}
+
+public interface IHandler
+{
+    bool Test<T>();
+}
+
+public class TypeChecker : ITypeChecker
+{
+    public static bool Test<T>() => true;
+}
+
+public class Handler<TChecker> : IHandler where TChecker : ITypeChecker
+{
+    public bool Test<T>() => TChecker.Test<T>();
+}
+
+public class test
+{
+    [Fact]
+    public static int TestEntryPoint()
```

Comment:
This test seems quite trivial. Would it even fail on CI ? Or is the failure dependent on compilation flags that we don't use on CI.

Reply:
The issue occurred in full AOT mode only. This test is used locally to reproduce and fix the issue, which is similar to the customer reported issue.

Reply:
If you disable your fix will this test fail on CI currently ?

Reply:
Before adding a new test, we should verify if it is not already covered on the CI. This will be done once the AOT job is fixed. The customer issue won't be closed until we verify that this case has test coverage.